README.md

```diff
@@ -97,12 +97,28 @@ Derek edit title: Question - does this work on Windows 10?
 
 Labels can be used to triage work or help sort it.
 
+To add a label:
 ```
 Derek add label: proposal
-Derek add label: help wanted
+```
+
+and to remove:
+```
 Derek remove label: bug
 ```
 
+To address multiple labels through a single action use a comma separated list.
+
+To add multiple labels:
+```
+Derek add label: proposal, help wanted, skill/intermediate
```

Comment:
Can this work both with and without the space?

Reply:
Yes, its splits and then trims.  If there's no space then trim's output is the same as the input.